tmt/options.py

```diff
@@ -183,12 +183,8 @@ def option(
 
 WORKDIR_ROOT_OPTIONS: list[ClickOptionDecoratorType] = [
     option(
-        '--workdir-root', metavar='PATH', envvar='TMT_WORKDIR_ROOT',
-        default=tmt.utils.WORKDIR_ROOT,
-        help=f"""
-             Path to root directory containing run workdirs.
-             Defaults to '{tmt.utils.WORKDIR_ROOT}'.
-             """)
+        '--workdir-root', metavar='PATH',
+        help='Path to root directory containing run workdirs.'),
```

Comment:
I'd argue the default value and envvar should be still used, `TMT_WORKDIR_ROOT` would still be considered as "workdir root", and so on. Is there a reason to remove it?

Reply:
I think we have effective_workdir_root to take care of the default now, but sure, we can also add the default value here:)

Reply:
And the help text? There is no reason to remove mention of default value from help, is it?